lib/vanagon/project.rb

```diff
@@ -286,9 +297,48 @@ def get_preinstall_actions(pkg_state)
       end
     end
 
+    # Collects the install trigger scripts for the project for the specified packing state
+    #
+    # @param pkg_state String the package state we want to run the given scripts for.
+    #  Can be one or more of the 'install' or 'upgrade'
+    # @return {} of scriptlets to execute during the pkg_state (install or upgrade)
+    #   there can be more than one script for each package (key)
```

Comment:
The Yardoc is malformed. Try:

```
# @param [String] pkg_state the package state we want to run the given scripts for.
#   Can be one or more of the 'install' or 'upgrade'
# @return [Hash] a collection of one or more scriptlets to execute during the pkg_state (install or upgrade)
```

Reply:
FYI, [this is a handy cheatsheet reference](https://gist.github.com/chetan/1827484)